themes/beaver/layouts/partials/page/site-scripts.html

```diff
@@ -1,12 +1,10 @@
 <script defer src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
-<script defer
-        src='{{ (resources.Get "js/bb-theme/pp_advanced_menu.js" | js.Build (dict "minify" true) | minify | fingerprint | resources.PostProcess).RelPermalink }}'>
-</script>
 
 {{
 $js := slice
   (resources.Get "js/587.js")
   (resources.Get "js/bb-theme/pp_advanced_tabs.js")
+  (resources.Get "js/navigation.js")
```

Comment:
make it as `js/bb-theme/pp_advanced_menu.js` as separate js, because it's critical and will be everywhere

Reply:
```html
<script defer
        src='{{ (resources.Get "js/bb-theme/pp_advanced_menu.js" | js.Build (dict "minify" true) | minify | fingerprint | resources.PostProcess).RelPermalink }}'>
</script>
```

Reply:
but it's maybe ok for now